dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -95,14 +105,18 @@ void LocalAdmissionController::startBackgroudJob()
             else
                 fetchTokensForLowTokenResourceGroups();
 
-            if (local_refill_token_callback)
-                local_refill_token_callback();
+            {
+                // Need lock here to avoid RCQ has already been destroied.
+                std::lock_guard lock(mu);
+                if (refill_token_callback)
+                    refill_token_callback();
```

Comment:
Why does this happen? Hasn't unregisterCallback been called before RCQ destroied?

Reply:
The original implement use `local_refill_token_callback`, so this can happen:
1. local_refill_token_callback copied from `LAC::refill_token_callback`
2. LAC::unregisterRefillTokenCallback is called
3. RCQ is destroied
4. local_refill_token_callback is still called

Reply:
ok